modules/programs/zsh/zsh-abbr.nix

```
@@ -36,7 +49,9 @@ in {
     xdg.configFile = {
       "zsh-abbr/user-abbreviations".text = concatStringsSep "\n"
         (mapAttrsToList (k: v: "abbr ${escapeShellArg k}=${escapeShellArg v}")
-          cfg.abbreviations) + "\n";
+          cfg.abbreviations ++ mapAttrsToList
+          (k: v: "abbr -g ${escapeShellArg k}=${escapeShellArg v}")
+          cfg.globalAbbreviations) + "\n";
```

Could be more readable this way:

```suggestion
      "zsh-abbr/user-abbreviations".text = 
      let
        abbreviations = mapAttrsToList
          (k: v: "abbr ${escapeShellArg k}=${escapeShellArg v}")
          cfg.abbreviations;
        globalAbbreviations = mapAttrsToList
          (k: v: "abbr -g ${escapeShellArg k}=${escapeShellArg v}")
          cfg.globalAbbreviations;
        allAbbreviations = abbreviations ++ globalAbbreviations;
      in
      concatStringsSep "\n" allAbbreviations + "\n";

```

---

I agree. However, when I followed your instructions, the format script broke the structure. So, I moved `let...in` block. How does this look? f8e5f87